docs/command-reference/stream/xadd.md

```diff
@@ -1,114 +1,120 @@
 ---
 description:  Learn how to use Redis XADD to append a new entry to a stream.
 ---
-
 import PageTitle from '@site/src/components/PageTitle';
 
 # XADD
 
 <PageTitle title="Redis XADD Command (Documentation) | Dragonfly" />
 
-## Introduction
-
-In Dragonfly, as well as in Redis and Valkey, the `XADD` command is used to append a new entry to a stream. 
-Streams are data structures that enable storing and processing ordered logs of events, and `XADD` is essential for adding data to these streams.
-
 ## Syntax
 
-```shell
-XADD key [<MAXLEN | MINID> [~ | =] threshold] <* | id> field value [field value ...]
-```
-
-## Parameter Explanations
+    XADD key [NOMKSTREAM] [<MAXLEN | MINID> [= | ~] threshold [LIMIT count]] <* | id> field value [field value ...]
```

Comment:
Streams docs were revamped... Changes in the `streams` directory should be unchecked, I suppose you moved those files from `pubsub` over here. 😃 They should be deleted instead.